components/recurring-contributions/RecurringContributionsPopUp.js

```diff
@@ -59,10 +62,34 @@ const MenuSection = styled(Flex).attrs({
   width: 1,
 })``;
 
+const i18nRadioCopy = (option, intl) => {
+  let msg;
+  switch (option) {
+    case 'No longer want to back the collective':
+      msg = intl.formatMessage({
+        id: 'subscription.cancel.reason1',
+        defaultMessage: 'No longer want to back the collective',
+      });
+      return msg;
+    case 'Changing payment method or amount':
+      msg = intl.formatMessage({
+        id: 'subscription.cancel.reason2',
+        defaultMessage: 'Changing payment method or armount',
+      });
+      return msg;
+    default:
+      msg = intl.formatMessage({
+        id: 'subscription.cancel.other',
+        defaultMessage: 'Other',
+      });
+      return msg;
+  }
+};
+
```

Comment:
Applied your suggestions.  You're right, there isn't text that can be internationalized in the component.  More so I was thinking that it would be nice if you have added id's in your lang.json files that you refer to, to have the internationalization actually take that as a prop and internationalize for you that way.  After thinking about it more, idk if it would actually be possible given how strict the library can be about passing static text.  Decided not resolve this in case you wanted to review/comment on it more.